src/main.cpp

```diff
@@ -77,6 +85,8 @@ int main() {
             DrawCylinder(p.pos, PLAYER_RADII-0.1, PLAYER_RADII, PLAYER_HEIGHT, PLAYER_RES, p.getColor());
         }
 
+        if (IsKeyUp(KEY_W)) walk.Play();
```

Comment:
shouldn't this be `IsKeyDown()`?

Reply:
for some reason, `IsKeyDown` plays the sound when the key is released, not when it's pressed